submittal/src/main/java/com/kyriba/submittal/api/ConfirmationController.java

```diff
@@ -0,0 +1,48 @@
+package com.kyriba.submittal.api;
+
+import com.kyriba.submittal.service.SpeakerService;
+import lombok.Builder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+
+
+/**
+ * @author M-ABL
+ */
+@RestController
+@RequestMapping(value = "/api/v1/confirm",
+    consumes = APPLICATION_JSON_UTF8_VALUE,
+    produces = APPLICATION_JSON_UTF8_VALUE)
+public class ConfirmationController
+{
+  private final SpeakerService speakerService;
+
+
+  @Autowired
+  ConfirmationController(final SpeakerService speakerService)
+  {
+    this.speakerService = speakerService;
+  }
+
+
+  @GetMapping(path = "/registration")

```

Comment:
On GET request there must be no active business logic, just reading. And resource URL must be identifiable.

Reply:
and what's the best option for the confirmation request? POST?

Reply:
POST and PUT are modification commands, GET is not and idempotent.

Reply:
Body is the primary source of information for POST command. Sometimes headers are also used.